pkg/api/v1beta3/conversion_generated.go

```diff
@@ -6692,6 +6866,7 @@ func init() {
 		autoConvert_api_ObjectReference_To_v1beta3_ObjectReference,
 		autoConvert_api_Parameter_To_v1beta3_Parameter,
 		autoConvert_api_PersistentVolumeClaimVolumeSource_To_v1beta3_PersistentVolumeClaimVolumeSource,
+		autoConvert_api_PodSecurityContext_To_v1beta3_PodSecurityContext,
```

Comment:
also interesting, this is still generated and registered but not used.  Same thing occurs for the other custom conversion functions in v1beta3/conversion.go